### PR TITLE
Feature/finish api routes

### LINKED ISSUE #261: When a setting is changed, it's not being saved
https://github.com/watermelontools/watermelon/assets/8325094/1302a35f-b759-499f-b222-686615bc0eeb

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Description
An effort to not use pages router no mo'
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)  
- [x] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update  
- [ ] Chore: cleanup/renaming, etc  
- [ ] RFC  
## Notes
Will allow new metrics
Fixes #261 
## Acceptance

- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by baristaGeek** [comment]
Let's wait after Show HN for this one

**Comment by EstebanDalelR** [comment]
> Let's wait after Show HN for this one

Why?
What's the risk?

File: app/api/hover/getHoverData/route.ts
Comment (baristaGeek) [inline app/api/hover/getHoverData/route.ts:1]: Ok this is good but let's touch base on this during our next planning session.

**Review by baristaGeek — APPROVED** [review]

File: app/api/hover/getHoverData/route.ts
Comment (EstebanDalelR) [reply app/api/hover/getHoverData/route.ts:1]: So we went through planning, but no mention.

**Pull request merged**